src/emmy/calculus/derivative.cljc

```diff
@@ -396,7 +397,7 @@
          df (d f)
          df* (d (fn [args] (apply f args)))]
      (-> (fn
-           ([] (constantly 0))
```

Comment:
@littleredcomputer this was a bug! for a no-arg `f`, `((D f))` should return 0, not another function.